charts/consul/hack/mass-pr-commenter/README.md

```diff
@@ -0,0 +1,25 @@
+# Mass PR Commenter Script
+
+This script allows the user to comment on every open pull request in a given repository.
+
+## Prerequisites
```

Comment:
:driveby-comment: I think the `gh` CLI is another pre-requisite?

Reply:
Yes! It is.